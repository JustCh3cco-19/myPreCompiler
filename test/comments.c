// questo è un commento a linea
int a; /* commento dopo il codice */
char *s = "stringa // non-commento";
/*
   commento 
   su più righe
*/
