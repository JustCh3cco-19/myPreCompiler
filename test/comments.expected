int a;
char *s = "stringa // non-commento";
